etk/extractors/date_extractor.py

```diff
@@ -15,6 +15,7 @@ def __init__(self, ignore_future_dates: bool=True, ignore_past_years: int=40) ->
         self.ignore_past_years = ignore_past_years
 
     def parse_date(self, str_date: str, settings: dict=None) -> datetime.datetime or None:
```

Comment:
parse_date is a separate extractor. The difference is that date_extractor scans the text to find dates. parse_date tests whether its input is a date.